SS14.Launcher/Views/MainWindowTabs/ServerEntryView.xaml

```diff
@@ -21,6 +21,7 @@
                      Margin="10, 0" />
           <!-- TODO: Enable text trimming here when Avalonia 0.10 is out of preview and we switch to it. -->
           <TextBlock VerticalAlignment="Center" Text="{Binding Name}" />
+          <TextBlock VerticalAlignment="Center" TextAlignment="Right" Text="{Binding ForkID}" />
```

Comment:
This is way too much clutter to fit in the header line.

Reply:
yes, the UI is the most unfinished part of this PR, I am waiting on the other 2 PRs